conf/README.md

```diff
@@ -1,26 +0,0 @@
-# What is this for?
```

Comment:
I would prefer updating this file over removing it. It's not so out-of-date after all.